cosmoz-default-tree.js

```diff
@@ -0,0 +1,296 @@
+import { CosmozTree } from './cosmoz-tree.js';
+
+/**
+Navigator through object with treelike datastructure and default settings.
+
+@demo demo/index.html
+*/
+export class CosmozDefaultTree extends CosmozTree {
+
+	/**
+	 * @param {Object} treeData (The tree object.)
+	 * @param {Object} options (Tree options.)
+	 * @param {String} options.childProperty ["children"] (The name of the property a search should be based on. e.g. "name")
+	 * @param {String} options.propertyName ["name"] (The name of the property a search should be based on. e.g. "name")
+	 * @param {String} options.pathStringSeparator ["/"] (The string the path should get separated with.)
+	 * @param {String} options.pathLocatorSeparator ["."] (The string which separates the path segments of a path locator.)
+	*/
+	constructor(treeData, options = {}) {
+		super();
+		this._treeData = treeData;
+		this._roots = CosmozDefaultTree._objectValues(treeData);
+
+		this.pathLocatorSeparator = options.pathLocatorSeparator || '.';
+		this.pathStringSeparator = options.pathStringSeparator || '/';
+		this.childProperty = options.childProperty || 'children';
+		this.searchProperty = options.searchProperty || 'name';
+	}
+
+	static _objectValues(obj) {
+		if (!obj) {
+			return [];
+		}
+		return Object.keys(obj).map(key => obj[key]);
```

Comment:
```suggestion
		return Object.values(obj);
```

orv maybe drop the helper completely?